fuel/transformers/__init__.py

```diff
@@ -271,12 +395,17 @@ class Cache(Transformer):
         refilled when needed through the :meth:`get_data` method.
 
     """
-    def __init__(self, data_stream, iteration_scheme):
+    def __init__(self, data_stream, iteration_scheme, **kwargs):
```

Comment:
You're right about axis labels, we can safely assume they won't change, as the effect of `Cache` is to change the batch size of a data stream that produces batches.